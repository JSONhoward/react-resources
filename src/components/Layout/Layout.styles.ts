import styled from 'styled-components';

export const StyledLayout = styled('div')`
position: relative;
display: flex;
min-height: 100vh;
flex-direction: column;
`