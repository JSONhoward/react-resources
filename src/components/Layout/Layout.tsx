import React from 'react'

import { StyledLayout } from './Layout.styles';

const Layout: React.FC = ({children}) => {
    return (
        <StyledLayout>
            {children}
        </StyledLayout>
    )
}

export default Layout
