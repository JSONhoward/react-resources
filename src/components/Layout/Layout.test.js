import React from 'react'
import {create} from 'react-test-renderer'
import Layout from './'

describe('Layout Tests', () => {
    test('Snapshot Test', () => {
        const tree = create(<Layout />).toJSON()

        expect(tree).toMatchSnapshot()
    })
})