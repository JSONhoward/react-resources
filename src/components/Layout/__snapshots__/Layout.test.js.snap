// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Tests Snapshot Test 1`] = `
<div
  className="sc-bdnylx lbThmN"
/>
`;
